rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/aws/AWSBakeHandlerSpec.groovy

```diff
@@ -354,6 +358,71 @@ class AWSBakeHandlerSpec extends Specification {
       1 * packerCommandFactoryMock.buildPackerCommand("", parameterMap, "$configDir/$awsBakeryDefaults.templateFile")
   }
 
+  void 'produces packer command with all required parameters for amzn, with sudo using runAsRoot:true'() {
+    setup:
+      def awsBakeryDefaultsJson = [
+        awsAccessKey: "FOO",
```

Comment:
we removed the aws keys from the rosco.bakeryDefaults last week
https://github.com/spinnaker/spinnaker/commit/8a2da11c9d29e3a879a068c197568c108b9e5f88